src/components/20-molecules/top-content-bar/index.scss

```diff
@@ -31,6 +31,16 @@
       }
     }
   }
+
+  &__children {
+    display: flex;
+    align-items: center;
+
+    .m-top-content-bar__button-link,
+    .m-top-content-bar__button {
```

Comment:
Isnt it possible to use `.m-top-content-bar__*`? To put this margin to all childs?

Reply:
No, we would need a child css-template just for that. And in child css-templates we cannot reference classes.